src/components/SecondaryPanes/CommandBar.js

```diff
@@ -98,6 +97,21 @@ function debugBtn(onClick, type, className, tooltip, disabled = false) {
 }
 
 class CommandBar extends Component {
+  props: {
+    sources: Object,
+    selectedSource: Object,
+    resume: () => any,
+    stepIn: () => any,
+    stepOut: () => any,
+    stepOver: () => any,
+    breakOnNext: () => any,
+    pause: Map<string, any>,
```

Comment:
![image](https://cloud.githubusercontent.com/assets/16179366/26087193/07cf38e6-39b5-11e7-8da6-bb07e5a3d682.png)

In the Pause reducer, the 'pause' prop is not of type Map. Does this mean I need to convert this to a Map (like Map<string, Pause> )or should I change the above file (CommandBar)ile to just 'Pause' object instead of Map type?